report/src/presence_intro.tex

```diff
@@ -0,0 +1,23 @@
+\section{\label{sec:presence_intro}Room Occupancy Detection for Meeting Start and End Times}
+
+To address the problem defined in section \ref{TODO-SECTION 2.a.i AND 2.b.iii}, we propose an occupancy detection system (presence system) to identify when a meeting starts, defined as participants entering a room, and when it ends, defined as all participants leaving the meeting room.
+This section investigates various technologies for detecting room occupancy, evaluates their performance in the context of meeting start and end times, and determines the most suitable technology for our context of use.
+This technology will serve as the basis for the rest of the system, which will be described in the following sections.
+
+Several technologies can be utilized for detecting room occupancy, including infrared (IR) sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, ultrasonic sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, microwave sensors\cite{woodward-2021}, computer vision\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{OccupancySensorMotion}, Bluetooth and Wi-Fi\cite{teissedre-2019}, pressure sensors\cite{OccupancySensorMotion}, acoustic sensors\cite{OccupancySensorMotion}, and CO2 sensors\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{jinSensingProxyOccupancy2015}.\cite{faragherLocationFingerprintingBluetooth2015}\todo{Insert all the types you find and cite here. Remove TODO when this section is done.}
+Each technology has its strengths and weaknesses in terms of accuracy, cost, and performance.
+
+We have chosen accuracy, responsiveness, privacy, cost and environmental factors as evaluation criteria for occupancy detection technologies because they cover the key aspects necessary for successful implementation.
+Accuracy and responsiveness ensure reliable detection and efficient space management, while privacy addresses user confidence and regulatory compliance.
+Cost ensures affordability and sustainability, and environmental factors guarantee consistent performance under varying conditions.
+Taken together, these criteria help to select a technology that fits the context of use.
+
+To determine a suitable technology, we will now evaluate the technologies based on the criteria listed above.
+
+% \begin{itemize}
+%     \item Accuracy: The ability to correctly detect the presence or absence of people in the room
+%     \item Responsiveness: The ability to promptly detect changes in room occupancy
+%     \item Privacy: Ensuring that the detection method respects the privacy of meeting participants
+%     \item Cost: The expenses associated with acquiring, installing, and maintaining the technology
+%     \item Environmental factors: The technology's robustness to various environmental conditions
+% \end{itemize}
```

Comment:
Do we want a listing or not?

```suggestion
```

Reply:
A listing would be good here. 
We could say "Evaluation of presence technologies can be based on criteria seen in listing xxx"
However, we still need sources and better reasoning for why these criteria are meaningful. Why is accuracy necessary? Why is privacy something that matters? ....

Reply:
Added listing